src/test/fuzz/script_ops.cpp

```diff
@@ -18,10 +18,10 @@ void test_one_input(const std::vector<uint8_t>& buffer)
     while (fuzzed_data_provider.remaining_bytes() > 0) {
         switch (fuzzed_data_provider.ConsumeIntegralInRange(0, 7)) {
         case 0:
-            script += ConsumeScript(fuzzed_data_provider);
+            (void)ConsumeScript(fuzzed_data_provider);
             break;
         case 1:
-            script = script + ConsumeScript(fuzzed_data_provider);
+            (void)ConsumeScript(fuzzed_data_provider);
```

Comment:
I think if you'd merge these two into one call for both case 0 & case 1, the fuzzer might work slightly more efficiently by not making 0 and 1 take a different code path.

Reply:
> I think if you'd merge these two into one call for both case 0 & case 1, the fuzzer might work slightly more efficiently by not making 0 and 1 take a different code path.

Tend to agree. If not going to collapse now could at least add a comment to avoid confusion here

Reply:
Isn't the point here not to invalidate the existing seed corpus?

Reply:
I mean *keep* case 0 and case 1, but merge them into a single branch.

Reply:
I am planning to replace them by `operator=(const&` and `operator=(&&` respectively

Reply:
Replaced  by `operator=(const&` and `operator=(&&` respectively